scss/_variables.scss

```diff
@@ -738,6 +738,7 @@ $navbar-dark-brand-hover-color:           $navbar-dark-active-color !default;
 $dropdown-min-width:                10rem !default;
 $dropdown-padding-y:                .5rem !default;
 $dropdown-spacer:                   .125rem !default;
+$dropdown-font-size:                $btn-font-size !default;
```

Comment:
Should use `$font-size-base` because this font size is used not only for dropdown item button(link) but also text and forms: https://getbootstrap.com/docs/4.2/components/dropdowns/#text